aggregator/src/main/java/com/redhat/cloud/notifications/config/AggregatorConfig.java

```diff
@@ -34,7 +34,7 @@ private static String toggleName(String feature) {
     @Deprecated(forRemoval = true, since = "To be removed when we're done migrating to Unleash in all environments")
     boolean unleashEnabled;
 
-    @ConfigProperty(name = "notifications.bundle.level.digest.enabled", defaultValue = "false")
+    @ConfigProperty(name = "notifications.bundle.level.digest.enabled", defaultValue = "true")
```

Comment:
Could you please update the ClowdApp template as well?

Reply:
Sure